src/screens/ConnectSignUpScreen.js

```diff
@@ -120,6 +122,23 @@ export default class ConnectSignUp extends React.Component {
     }
   }
 
+  getFormOptions() {
+    return {
+      fields: {
+        phoneNumber: {
+          help: 'Disclaimer: Addresses will not be shared without permission from users',
+          hasError: !!this.state.errors.first_name,
+          error: this.state.errors.first_name,
```

Comment:
This field should correspond to the rails version (snake case) of the property name, which is phone number in this case: 

```javascript
hasError: !!this.state.errors.phone_number,
error: this.state.errors.phone_number,
```